django/applications/catmaid/static/js/widgets/landmark-widget.js

```diff
@@ -1707,8 +1780,12 @@
             });
         };
 
-        $('div.dataTables_length', landmarkDataTable.table().container())
-            .append(deleteSelected);
+        tableContainer.append(deleteSelected);
+
+        const exportSelected = document.createElement('button');
+        exportSelected.appendChild(document.createTextNode('Export selected as CSV'));
+        exportSelected.onclick = function () { widget.exportLandmarks(); };
```

Comment:
Could probably be done better; wasn't sure how `this` binding would work if I just passed the instance method.